python/brayns/plugins/bbp/bbp_loader.py

```diff
@@ -24,17 +24,15 @@
 from brayns.core.model.model_loader import ModelLoader
 from brayns.plugins.bbp.bbp_cells import BbpCells
 from brayns.plugins.bbp.bbp_report import BbpReport
+from brayns.plugins.common.morphology_parameters import MorphologyParameters
 
 
 @dataclass
 class BbpLoader(ModelLoader):
 
     cells: BbpCells = BbpCells.all()
     report: Optional[BbpReport] = None
-    radius_multiplier: float = 1.0
-    load_soma: bool = True
-    load_axon: bool = False
-    load_dendrites: bool = False
+    morphology: Optional[MorphologyParameters] = MorphologyParameters()
```

Comment:
The problem with this is that your default value is mutable.

As default values are initialized when the module is loaded, all default constructed instances of BbpLoader will shared the same instance of the params.

This can lead to weird errors and is considered as bad practice.

Possible solutions would be to mark your params as frozen (@dataclass(frozen=True)) or to set it to None by default and fill the default parameters on the C++ side.

Also having the parameters unpacked in the loader constructor made the syntax a bit easier IMO but I guess it is also fine like that.